### PR TITLE
Acceptance tests

### PR DESCRIPTION
**Pull request by supertommc**
Added acceptance tests and also some screenshorts from the tests' output

### CODE REVIEW

File: retrospective_wall/test_driver/steps/bubble_creation_steps.dart [hunk 1]
```diff
@@ -0,0 +1,140 @@
+import 'package:flutter_driver/flutter_driver.dart';
+import 'package:flutter_gherkin/flutter_gherkin.dart';
+import 'package:gherkin/gherkin.dart';
+import 'package:retrospective_wall/globals.dart';
+
+class CheckGivenWidgets extends Given1WithWorld<String, FlutterWorld> {
+  @override
+  Future<void> executeStep(String input1) async {
+    print("Test Given!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!");
```
Comment (JoaoASousa) [inline retrospective_wall/test_driver/steps/bubble_creation_steps.dart]: Remove Comment!

**Review by JoaoASousa — COMMENTED** [review]
Found 1 unwanted print.

File: retrospective_wall/test_driver/steps/bubble_creation_steps.dart (hunk 1 above)
Comment (supertommc) [reply retrospective_wall/test_driver/steps/bubble_creation_steps.dart]: Fixed! Thanks for the feedback

**Review by JoaoASousa — APPROVED** [review]
Looks good to me now!

**Pull request merged**